contrib/database/sql/conn.go

```diff
@@ -303,7 +303,12 @@ func (tp *traceParams) tryTrace(ctx context.Context, qtype QueryType, query stri
 	if !math.IsNaN(tp.cfg.analyticsRate) {
 		opts = append(opts, tracer.Tag(ext.EventSampleRate, tp.cfg.analyticsRate))
 	}
+	//ctx = WithSpanTags(ctx, tp.meta)
+	var cur_span, _ = tracer.SpanFromContext(ctx)
 	span, _ := tracer.StartSpanFromContext(ctx, tp.cfg.spanName, opts...)
+	cur_span.SetTag(ext.TargetHost, tp.meta[ext.TargetHost])
+	cur_span.SetTag(ext.TargetPort, tp.meta[ext.TargetPort])
+	cur_span.SetTag(ext.DBName, tp.meta[ext.DBName])
```

Comment:
This approach currently works because the test is creating a parent span, which is not gonna be always the case. Also, its probably not good to modify the parent span with these unrelated tags (imagine the parent span is an http one that has a different value for any of these tags).

I proposed an alternative in the review comment, please take a look!